scripts/update-size.js

```diff
@@ -1,17 +1,15 @@
 const fs = require('fs');
 const path = require('path');
-const formatBytes = require('format-bytes');
 const sizeLimit = require('size-limit');
-
 const root = path.join(__dirname, '..');
 const readmePath = path.join(root, 'README.md');
 const readme = fs.readFileSync(readmePath, 'utf-8');
 const sizeConfig = require('../.size-limit')[0];
-
+const byte = require('bytes');
```

Comment:
while this work bidirectionally I find it confusing, would prefer using more explicit parse and format functions, exported by this package